CHANGELOG.md

```diff
@@ -238,6 +238,16 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
   removed
 - The deprecated `@cumulus/common/workflows.getWorkflowTemplate` function has
   been removed
+- `@cumulus/aws-client/StepFunctions.toSfnExecutionName()`
```

Comment:
Nit:  we should proably s/has been removed//g from this block for consistency.  I agree repeating it is redundant though.

Reply:
Agreed. There are a number of other formatting issues in the Changelog that need to be addressed, but I was going to issue a separate PR to fix that document.